extensions/blocks/donations/upgrade-plan.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable wpcalypso/import-docblock */
```

Comment:
We may be able to move away from this if we switch to relying on existing tools to mark the block as paid. When we do so, we'll benefit from `wrapPaidBlock` out of the box.